Python 3.14 made functools.partial a method descriptor (cpython 121027),
so these class attributes began binding self, breaking unpickling of any
save or persistent file containing a datetime. Backport of the upstream
fix (renpy commit a0efe3508), first released after 8.5.3.

Index: renpy/compat/pickle.py
--- renpy/compat/pickle.py.orig
+++ renpy/compat/pickle.py
@@ -272,9 +272,9 @@ def make_datetime(cls, *args, **kwargs):
 
 
 class Unpickler(pickle.Unpickler):
-    date = functools.partial(make_datetime, datetime.date)
-    time = functools.partial(make_datetime, datetime.time)
-    datetime = functools.partial(make_datetime, datetime.datetime)
+    date = staticmethod(functools.partial(make_datetime, datetime.date))
+    time = staticmethod(functools.partial(make_datetime, datetime.time))
+    datetime = staticmethod(functools.partial(make_datetime, datetime.datetime))
 
     def find_class(self, module, name):
         if module == "datetime":
